lint.whitelist

```diff
@@ -475,9 +471,6 @@ CSS-COLLIDING-SUPPORT-NAME: css/*/LOCK
 # These are all the current "merge mismatch" errors the build system produces
 CSS-COLLIDING-SUPPORT-NAME: css/css-flexbox/support/200x200-green.png
 CSS-COLLIDING-SUPPORT-NAME: css/css-grid/grid-items/support/200x200-green.png
-CSS-COLLIDING-SUPPORT-NAME: css/css-scroll-snap/support/common.css
```

Comment:
was this intentional?

Reply:
Yes, this lint lists all of the colliding names, and if only one was left after my initial edits I removed that one too. The lint still passes :)